### ISSUE TITLE
EC_number is an invalid GFF3 attribute key

### ISSUE THREAD

**Issue by standage** [tseemann/prokka#443]
The [GFF3 specification](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md) states:

> *All attributes that begin with an uppercase letter are reserved for later use. Attributes that begin with a lowercase letter can be used freely by applications.*

"Reserved for later use" is generally interpreted to mean they can only be defined by the official spec. Indeed, when I sort and tidy Prokka output with [gt gff3](http://genometools.org/tools/gt_gff3.html), I get a bunch of the following error messages.

```
warning: illegal uppercase attribute "EC_number" on line 83 in file "prokka-outdir/prokka.gff"; change to lower case
```

GenomeTools converts the attribute key from `EC_number` to `eC_number`.

**Comment by tseemann** [comment]
Yes, I violate the GFF3 spec for compatibility with INSDC feature tag standard:
http://www.insdc.org/documents/feature_table.html
I'm not sure what the best approach is. 
I have other logic in the code to ensure `EC_number` is maintained too.

You can do this in your pipeline if needed
`sed s/EC_number/ec_number/g` < broken.gff > fixed.gff`
